From e81ba3164eae9a8fe80d38c97b96f2314a319539 Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 22:43:09 -0400
Subject: [infiniband] cxgb4: Use pr_warn_ratelimited

Message-id: <1410475447-94392-51-git-send-email-svemuri@redhat.com>
Patchwork-id: 93868
O-Subject: [RHEL7.1 PATCH BZ 1124947 050/108] RDMA/cxgb4: Use pr_warn_ratelimited
Bugzilla: 1124947
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit aec844df104f1e45cafd10628481f256908554c4)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/cxgb4/resource.c b/drivers/infiniband/hw/cxgb4/resource.c
index cdef4d7..94b5fd9 100644
--- a/drivers/infiniband/hw/cxgb4/resource.c
+++ b/drivers/infiniband/hw/cxgb4/resource.c
@@ -322,8 +322,8 @@ u32 c4iw_rqtpool_alloc(struct c4iw_rdev *rdev, int size)
  unsigned long addr = gen_pool_alloc(rdev->rqt_pool, size << 6);
  PDBG("%s addr 0x%x size %d\n", __func__, (u32)addr, size << 6);
  if (!addr)
-  printk_ratelimited(KERN_WARNING MOD "%s: Out of RQT memory\n",
-         pci_name(rdev->lldi.pdev));
+  pr_warn_ratelimited(MOD "%s: Out of RQT memory\n",
+        pci_name(rdev->lldi.pdev));
  mutex_lock(&rdev->stats.lock);
  if (addr) {
   rdev->stats.rqt.cur += roundup(size << 6, 1 << MIN_RQT_SHIFT);
-- 
1.7.1